pkg/config/config_test.go

```diff
@@ -72,9 +72,9 @@ func TestValidateAdditionalResources(t *testing.T) {
 
 func TestValidateAdditionalResourcesFail(t *testing.T) {
 	testCases := [][]string{
-		[]string{"abcdef"},
-		[]string{""},
-		[]string{"test.v1.com"},
```

Comment:
Why remove types here?

Reply:
This is a redundant type from array composite literal, so doesn't need to be there.